packages/blade/src/components/ActionList/styles/StyledListBoxWrapper.native.tsx

```diff
@@ -5,8 +5,11 @@ import styled from 'styled-components/native';
 import { getBaseListBoxWrapperStyles } from './getBaseListBoxWrapperStyles';
 
 // It's not really styled here but have to keep name same since we have to style this with overflowY and maxWidth in web
-const StyledListBoxWrapper = styled(SectionList)((props) => ({
-  ...getBaseListBoxWrapperStyles({ theme: props.theme }),
+const StyledListBoxWrapper = styled(SectionList)<{ isInBottomSheet: boolean }>((props) => ({
+  ...getBaseListBoxWrapperStyles({
+    theme: props.theme,
+    isInBottomSheet: props.isInBottomSheet,
```

Comment:
There's two of the CSS props maxHeight and padding, if we make it accept just the object, we will have to maintain 

```
maxHeight: isInBottomSheet ? undefined : makeSize(size[300]),
padding: isInBottomSheet ? undefined : makeSize(props.theme.spacing[3]),
```

This code on both .web & .native file. Will be more flaky this way since we now have to remember to change in two files if we update tokens.